package.json

```diff
@@ -3,27 +3,28 @@
   "version": "1.0.0",
   "private": true,
   "devDependencies": {
-    "cross-env": "^5.1.4",
-    "dateformat": "^2.0.0",
-    "react-scripts": "1.1.4",
-    "cpy-cli": "^1.0.1"
+    "cpy-cli": "^2.0.0",
+    "cross-env": "^5.2.0",
+    "dateformat": "^3.0.3",
+    "react-scripts": "^2.0.0-next.a671462c"
   },
   "dependencies": {
-    "google-maps-react": "^1.1.2",
+    "@chevtek/react-spinners": "^1.0.5",
+    "google-maps-react": "^2.0.2",
     "immutable": "^3.8.2",
-    "kentico-cloud-delivery": "^4.0.2",
-    "lodash": "^4.17.4",
+    "kentico-cloud-delivery": "^4.3.0",
+    "lodash": "^4.17.10",
     "qs": "^6.5.2",
-    "react": "^15.3.1",
-    "react-cookie": "^2.2.0",
-    "react-dom": "^15.3.1",
-    "react-router-dom": "^4.2.2",
-    "react-scroll": "^1.6.4",
+    "react": "^16.4.2",
+    "react-cookie": "^3.0.4",
+    "react-dom": "^16.4.2",
+    "react-router-dom": "^4.3.1",
+    "react-scroll": "^1.7.10",
     "react-translate": "^6.0.0",
-    "rxjs": "^6.1.0",
-    "twitter-widgets": "^1.0.0",
-    "universal-cookie": "^2.2.0",
-    "validator": "^10.4.0"
+    "rxjs": "^6.2.2",
+    "twitter-widgets": "^2.0.0",
```

Comment:
There's some problem with the twitter widgets. If I navigate to: http://localhost:3000/en-us/articles/cf106f4e-30a4-42ef-b313-b8ea3fd3e5c5 (Articles->Coffee beverages explained) I get the following error:

![image](https://user-images.githubusercontent.com/9810625/44845219-3f582280-ac4d-11e8-926c-af898a2383a9.png)

Reply:
Oops - I've just committed a working version for version 2 of twitter-widgets.